components/count-badge/test/count-badge-icon.test.js

```diff
@@ -1,9 +1,135 @@
 import '../count-badge-icon.js';
-import { runConstructor } from '@brightspace-ui/testing';
+import { expect, fixture, html, runConstructor } from '@brightspace-ui/testing';
 
 describe('d2l-count-badge-icon', () => {
-	it('should construct', () => {
-		runConstructor('d2l-count-badge-icon');
+
+	describe('constructor', () => {
+		it('should construct', () => {
+			runConstructor('d2l-count-badge-icon');
+		});
+	});
+
+	describe('defaults', () => {
+
+		it('should default announceChanges to false', async() => {
+			const el = await fixture(html`<d2l-count-badge-icon icon="tier3:gear" number="1" text="1 notification"></d2l-count-badge-icon>`);
+			expect(el.announceChanges).to.be.false;
+		});
+
+	});
+
+	describe('number formatting', () => {
```

Comment:
This are likely covered by vdiffs but also test specific functionality of the component so I've left them.